lib/params/OpacityMap.cpp

```diff
@@ -620,3 +632,11 @@ double OpacityMap::maxValue() const {
 	double relmax = GetValueDouble(_relMaxTag, 1.0);
 	return (minmaxmap[0] + relmax*(minmaxmap[1]-minmaxmap[0]));
 }
+
+bool OpacityMap::GetDirtyBit() const {
+	return _dirtyBit;
+}
+
+void OpacityMap::ResetDirtyBit() {
```

Comment:
The name of Reset should be used when there's an apparent default value, so reset something clearly means it goes to the default value.
In this case, _dirtybit doesn't have an apparent default value, thus the name `ResetDirtyBit()` doesn't tell what this function does. 
Consider changing it to `SetDirtyBitFalse()` or taking in a value: `ResetDirtyBit( bool newValue )`.